spec/dummy/db/schema.rb

```diff
@@ -0,0 +1,85 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_05_22_195925) do
```

Comment:
Where is the migration that generates this?

Reply:
there you go: https://github.com/ManageIQ/manageiq-api-common/pull/91/commits/e2839f677c8cfc8039049369f66dc5deab02057c